README.md

```diff
@@ -137,6 +137,15 @@ so that we can verify your status.
 
 ## Non-validator nodes
 
+### Sidecars
+Each type of node has a set of required sidecars to operate in a stable manner (described in the table below).
+
+| Node      | Needs Ethereum Sidecar? (yes/no) | Needs Connect Sidecar? (yes/no) |
+|-----------|----------------------------------|---------------------------------|
+| Validator | yes                              | yes                             |
+| Seed Node | no                               | no                              |
+| RPC Node  | no                               | no                              |
```

Comment:
I would remove this section and just add a single sentence that non-validator nodes do not require the Ethereum and Connect sidecars.